### PR TITLE
Reland "Suppress unwanted warnings in googleapiclient.""

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/python/metrics/logs.py [hunk 1]
```diff
@@ -89,6 +96,7 @@ def get_handler_config(filename, backup_count):
 def get_logging_config_dict(name):
   """Get config dict for the logger `name`."""
   logging_handler = {
+      'appengine': get_handler_config('bot/logs/appengine.log', 1),
```
Comment (oliverchang) [inline src/python/metrics/logs.py]: why is this needed? what happens if we don't include it? 

Please include a comment if it's really needed :)

Comment (inferno-chromium) [reply src/python/metrics/logs.py]: Fixed tests to mock as if running in appengine. PTAL.
